pubpub/platform
# Configuring submission buttons

## Issue(s) Resolved
Closes https://github.com/pubpub/v7/issues/385

## Test Plan

* Open up a form in the form builder (or create a new one). You should see a button to add a button in the side panel
![image](https://github.com/user-attachments/assets/e3053898-b524-4a54-97aa-e7621e305236)

* Add a button and fill in fields.
* Click "Save" at the bottom of the form. This should take you back to the initial element panel, and you should now see your button
![image](https://github.com/user-attachments/assets/d517be49-5a63-4033-b999-8d4829a54005)

* Now you can either click the big save at the top which will persist to the db, or add another button
* There should be a validation error if you try to add a button with the same label 
![image](https://github.com/user-attachments/assets/7433363c-de4f-4aac-aa37-f805e91e53b6)


## Screenshots (if applicable)

https://github.com/user-attachments/assets/308f0f9d-a8db-4e6e-bfef-720e1a6104d0




## Optional

### Notes/Context/Gotchas

There is a state where there is no submit button configured, but I doubt anyone actually wants that. We could either make a default one that is created with each form, or have the external form always render a default submit button?

### Supporting Docs
